source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs

```diff
@@ -373,6 +373,25 @@ public void SubmitScore(string leaderboardId, long score, Action<bool> callback)
             }
         }
 
+        public void RequestPermissions(Action<RequestPermissionStatusCodes> callback, string[] scopes)
```

Comment:
It seems we should have a public doc here to be consistent

Reply:
done